irohad/main/impl/ordering_init.hpp

```diff
@@ -19,14 +19,18 @@
 #define IROHA_ORDERING_INIT_HPP
 
 #include "ametsuchi/peer_query.hpp"
+#include "logger/logger.hpp"
 #include "ordering/impl/ordering_gate_impl.hpp"
 #include "ordering/impl/ordering_gate_transport_grpc.hpp"
-#include "ordering/impl/ordering_service_transport_grpc.hpp"
-
-#include "logger/logger.hpp"
 #include "ordering/impl/ordering_service_impl.hpp"
+#include "ordering/impl/ordering_service_transport_grpc.hpp"
 
 namespace iroha {
+
+  namespace ametsuchi {
```

Comment:
Is there any way to avoid declaring this class here?

Reply:
forward declaration